x/programs/cmd/simulator/src/lib.rs

```diff
@@ -340,8 +340,7 @@ impl ClientBuilder<'_> {
         let Child { stdin, stdout, .. } = Command::new(self.path)
             .arg("interpreter")
             .arg("--cleanup")
-            .arg("--log-level")
-            .arg("error")
+            .args(["--log-level", "error"])
```

Comment:
It just makes our life easier since these two are so tightly coupled, it happens sometimes to write an arg in between and struggle to understand why the call to the simulator panics.